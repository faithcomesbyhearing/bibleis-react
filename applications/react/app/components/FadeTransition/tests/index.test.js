import React from 'react';
import renderer from 'react-test-renderer';
import FadeTransition from '..';

const children = (
	<div id="transition-child-id">
		<h1>Child of the transition</h1>
		<p>Paragraph child of the transition</p>
	</div>
);

describe('<FadeTransition /> component', () => {
	it('should match snapshot with classNames', () => {
		const tree = renderer
			.create(
				<FadeTransition classNames={'slide-from-right'}>
					{children}
				</FadeTransition>,
			)
			.toJSON();
		expect(tree).toMatchSnapshot();
	});
	it('should match snapshot without classNames', () => {
		const tree = renderer
			.create(<FadeTransition>{children}</FadeTransition>)
			.toJSON();
		expect(tree).toMatchSnapshot();
	});
});
