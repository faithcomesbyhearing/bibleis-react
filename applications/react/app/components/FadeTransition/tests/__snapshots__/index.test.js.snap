// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FadeTransition /> component should match snapshot with classNames 1`] = `
<div
  id="transition-child-id"
>
  <h1>
    Child of the transition
  </h1>
  <p>
    Paragraph child of the transition
  </p>
</div>
`;

exports[`<FadeTransition /> component should match snapshot without classNames 1`] = `
<div
  id="transition-child-id"
>
  <h1>
    Child of the transition
  </h1>
  <p>
    Paragraph child of the transition
  </p>
</div>
`;
